tutorials/migration_light_api.md

```diff
@@ -0,0 +1,144 @@
+* [`gazebo::physics::Light`](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1Light.html) - responsible for accessing and setting generic entity properties, and parent / child relationship.
+    * Example: [Light::GetName](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1Base.html#a9a98946a64f3893b085f650932c9dfee) / [Light::SetName](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1Entity.html#a5d74ac4d7a230aed1ab4b11933b16e92)
+    * Example: [Light::GetParent](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1Base.html#af87578478aeeb2b176de010d1d639fd9) / [Light::SetParent](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1Base.html#a736efe8278da4ecb1640100a2857756f)
+
+In Gazebo, the light APIs has been consolidated into a single Light class with
+some of generic functions available through other utility / core classes.
+You'll find the APIs below on the following headers:
+
+* [ignition/gazebo/Light.hh](https://gazebosim.org/api/gazebo/6/Light_8hh.html)
+* [ignition/gazebo/Util.hh](https://gazebosim.org/api/gazebo/6/Util_8hh.html)
+* [ignition/gazebo/SdfEntityCreator.hh](https://gazebosim.org/api/gazebo/6/SdfEntityCreator_8hh.html)
+* [ignition/gazebo/EntityComponentManager.hh](https://gazebosim.org/api/gazebo/6/classignition_1_1gazebo_1_1EntityComponentManager.html)
+
+It's worth remembering that most of this functionality can be performed using
+the
+[EntityComponentManager](https://gazebosim.org/api/gazebo/6/classignition_1_1gazebo_1_1EntityComponentManager.html)
+directly. The functions presented here exist for convenience and readability.
+
+### Properties
+
+Most of Gazebo-classic's Light API is related to setting
+and getting properties. These functions are great candidates to have
+equivalents on Gazebo because the Entity-Component-System architecture is
+perfect for setting components (properties) into entities such as lights.
+This section focuses on migrating from APIs provided through the
+`gazebo::rendering::Light` class.
+
+---
+
+Classic | Gazebo
+-- | --
+CastShadows | `ignition::gazebo::Light::CastShadows`
+Clone | TODO
+DiffuseColor | `ignition::gazebo::Light::DiffuseColor`
+Direction | `ignition::gazebo::Light::Direction`
+FillMsg | TODO
+Id | `ignition::gazebo::Light::Entity`
+LightType | `ignition::gazebo::Light::Type`
+Name | `ignition::gazebo::Light::Name`
+Position | `ignition::gazebo::Light::Pose`
+Rotation | `ignition::gazebo::Light::Pose``
+SetAttenuation | use `ignition::gazebo::Light::SetAttenuation\*`
```

Comment:
```suggestion
SetAttenuation | use `ignition::gazebo::Light::SetAttenuation`
```

Reply:
ah the `*` is supposed to be there (for wildcard match). Removed `\`. [4320596](https://github.com/gazebosim/gz-sim/pull/1931/commits/4320596ba96f6ab062d8cc2d52f12a561d3eefe5)